### ISSUE TITLE
Something wrong with the save dialog

### ISSUE THREAD

**Issue by beastie1** [OtterBrowser/otter-browser#1216]
The weird behavior happens in the save dialog (when saving through **Save Link Target As** or by clicking a link to a file that can't be viewed in the browser).

While the **Save** button in that dialog looks visually like it's the *default* button, pressing **Enter** *cancels* the download.

**Comment by beastie1** [comment]
Now pressing Enter does opens the dialog, but eventually Otter crashes whether you start to download the file or cancel it.

**Comment by Emdek** [comment]
@beastie1, yes, it seems that Enter is handled somewhere else too which invalidates `Transfer` object.

**Comment by Diablopc** [comment]
On openSUSE 42.3 with RC3 AppImage have no problems with this. All works fine.
I hope to check this using native builded binaries some later.
